src/app/cube/details/details.component.ts

```diff
@@ -116,6 +119,18 @@ export class DetailsComponent implements OnInit, OnDestroy {
 
       this.getLearningObjectRatings();
     } catch (e) {
+
+      /**
+       * TODO: change status to 404 when issue #149 is closed
+       * if server error is thrown, navigate to not-found page
+       */
+
+      if (e instanceof HttpErrorResponse) {
+        if (e.status === 500) {
+          this.router.navigate(['not-found']);
+          console.log('error handling is correct but the routing is not'); // joel's dumb log
```

Comment:
I'm not sure that I'm following the meaning of this log. Should it be here in production or was it for development?

Reply:
It was for development. I'll remove it.